quickstarts/microsoft.maps/maps-use-sas/prereqs/.settings.json

```diff
@@ -0,0 +1,4 @@
+{
```

Comment:
remove this file and update the template to support cross resource group scenarios

Reply:
> If the prereqs and the sample must be deployed to the same resource group add a file named .settings.json to the prereqs folder and put the following json snippet into the file (the comment is optional). Do this only if required by the sample, otherwise it may block customer deployment scenarios:

Not sure this is possible, the azuredeploy.json is deploying to the prereqs.azuredeploy.json Key Vault's nested resource type `secrets`, the secret and parent resource must exist in the same resource group deployment as `azuredeploy.json`. I am avoiding outputting the accountSasToken to prevent a bad habit of logging the actual secret in deployment activity logs.